index.html

```diff
@@ -5,14 +5,14 @@
         <meta charset = "UTF-8">
     </head>
     <body>
-    <div class="border">
+   
         <div class="heading">
 
             <div id="langDiv">
                 <label id="webLang">Select webpage language: </label>
                 <select id="pageLanguage" onchange="changeLang()">
-                    <option value="en_us">English</option>
-                    <option value="ga">Irish</option>
+                    <option id="en_us" value="en_us">English</option>
+                    <option id="ga" value="ga">Irish</option>
```

Comment:
Hi Kimaya,
HTML gets a little weird when two elements have the same id attribute.
Can we change the ids here to make it different for these two elements on lines 14 and 15?
Thanks,
Julian